src/main/java/org/jose/F.java

```diff
@@ -4,11 +4,16 @@
     public static void main(String[] args) {
 
         int h = 0;
+        int j = 0;
         System.out.println("HOLA,ADIOS");
         F jl = new F();
+        F bu = new F();
         long start = System.currentTimeMillis();
-        h = jl.fibonaccirecursivo(Integer.parseInt(args[0]));
-        System.out.println("Recursive: " + h + " in " + (System.currentTimeMillis() - start) + "ms");
+        // h = jl.fibonaccirecursivo(Integer.parseInt(args[0]));
+        j = bu.fibonaccibottom_up(Integer.parseInt(args[0]));
+        // System.out.println("Recursive: " + h + " in " + (System.currentTimeMillis() - start) + "ms");
```

Comment:
even if it was correct to remove this, which it isn't, you shouldn't have commented code. commented code clutters files with stuff that isn't executed anyway and, since the project is tracked in git, we can always see what was there before in the git history if we need at some point - no reason to have useless stuff lying around